src/platform/packages/shared/kbn-dev-utils/src/plugin_list/generate_plugin_list.ts

```diff
@@ -7,80 +7,59 @@
  * License v3.0 only", or the "Server Side Public License, v 1".
  */
 
-import Path from 'path';
-
+import { relative, resolve } from 'path';
 import normalizePath from 'normalize-path';
 import { REPO_ROOT } from '@kbn/repo-info';
 
-import { Plugins } from './discover_plugins';
+import type { Plugins } from './discover_plugins';
 
 const sortPlugins = (plugins: Plugins) => plugins.sort((a, b) => a.id.localeCompare(b.id));
 
-function* printPlugins(plugins: Plugins, includes: string[]) {
+function markdownEscape(snippet: string): string {
+  return snippet.replaceAll('\n', ' ').replaceAll('|', '\\|');
+}
+
+function* printMarkdownPlugins(plugins: Plugins) {
   for (const plugin of sortPlugins(plugins)) {
     const path = normalizePath(plugin.relativeReadmePath || plugin.relativeDir);
-    yield '';
-
-    if (plugin.readmeAsciidocAnchor) {
-      yield `|<<${plugin.readmeAsciidocAnchor}>>`;
-
-      includes.push(`include::{kibana-root}/${path}[leveloffset=+1]`);
-    } else {
-      yield `|{kib-repo}blob/{branch}/${path}[${plugin.id}]`;
-    }
-
-    yield plugin.relativeReadmePath === undefined
-      ? '|WARNING: Missing README.'
-      : `|${plugin.readmeSnippet}`;
-
-    yield '';
+    const readmeSnippet =
+      plugin.relativeReadmePath && plugin.readmeSnippet
+        ? markdownEscape(plugin.readmeSnippet)
+        : 'WARNING: Missing or empty README.';
+    yield `| [${plugin.id}]({kib-repo}blob/{branch}/${path}) | ${readmeSnippet} |`;
   }
 }
 
 export function generatePluginList(ossPlugins: Plugins, xpackPlugins: Plugins) {
-  const includes: string[] = [];
+  return `<!---
+This is an automatically generated file. Please do not edit directly.
+Instead, run the following from within the kibana repository:
 
-  return `////
+  node scripts/build_plugin_list_docs
 
-NOTE:
-  This is an automatically generated file. Please do not edit directly. Instead, run the
-  following from within the kibana repository:
-
-    node scripts/build_plugin_list_docs
-
-  You can update the template within ${normalizePath(
-    Path.relative(REPO_ROOT, Path.resolve(__dirname, __filename))
+You can update the template within ${normalizePath(
+    relative(REPO_ROOT, resolve(__dirname, __filename))
   )}
+-->
 
-////
-
-[[plugin-list]]
-== List of {kib} plugins
-
-[discrete]
-=== src/plugins
-
-[%header,cols=2*]
-|===
-|Name
-|Description
-
-${Array.from(printPlugins(ossPlugins, includes)).join('\n')}
+---
+mapped_pages:
+  - https://www.elastic.co/guide/en/kibana/current/plugin-list.html
+---
```

Comment:
Those should be the very first lines of the file (before the comments starting with `%`